jekyll-serve.sh

```diff
@@ -2,4 +2,6 @@
 
 set -e
 
+bundle exec jekyll build
+bundle exec pandoc -s --from html _site/terms/word.html --reference-doc=reference.docx -o Gruntwork-TermsOfService.docx && echo -e "\e[1;31mNew TOS Document (Gruntwork-TermsOfService.docx) File Generated"
```

Comment:
Add `echo` before this process explaining what's happening.

Reply:
Also, where does the docx file get written to? Putting it in the root folder seems a bit odd. Perhaps generate it into `/pages/terms` and update links to it accordingly?

Reply:
Moved files to cleaner location in 23470ed0c04a01ce87c5e9ade959b3ad1c7dcd20.